cmake/CompilerFlags.cmake

```diff
@@ -38,6 +35,8 @@ IF ( MSVC AND NOT ( "${CMAKE_CXX_COMPILER_ID}" STREQUAL "Intel" ) ) # Visual C++
     # ADDITIONAL DEBUG-MODE-SPECIFIC FLAGS
     ADD_CXX_DEBUG_DEFINITIONS("/Ob0") # Disable inlining
     ADD_CXX_DEBUG_DEFINITIONS("/RTCsu") # Runtime checks
+    ADD_CXX_DEBUG_DEFINITIONS("/fp:strict") # Floating point model
+    ADD_CXX_DEBUG_DEFINITIONS("/DMSVC_DEBUG") # Triggers code in main.cc to catch floating point NaNs
```

Comment:
Is the "D" supposed to be here? DMSVC_DEBUG

Line 54 shows: #ifdef MSVC_DEBUG

Reply:
Yes, based on [other examples](https://github.com/NREL/EnergyPlus/blob/develop/cmake/CompilerFlags.cmake#L31-L32), /D sets a #def variable, followed by the variable name. When in VS, the variable name is blue if it's set. When it's not set, the code inside the #ifdef is greyed out.

Reply:
Agree with this approach, it's /D on MSVC but -D on GCC-like compilers.  (https://docs.microsoft.com/en-us/cpp/build/reference/d-preprocessor-definitions?view=vs-2017)